src/api/spec/helpers/webui/project_helper_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+RSpec.describe Webui::ProjectHelper, type: :helper do
+  describe '#show_status_comment' do
+    skip
+  end
+
+  describe '#project_bread_crumb' do
+    skip
+  end
+
+  describe '#format_seconds' do
+    skip
+  end
+
+  describe '#rebuild_time_col' do
+    skip
+  end
+
+  describe '#short_incident_name' do
+    skip
+  end
+
+  describe '#patchinfo_rating_color' do
+    it 'returns the right color' do
+      expect(patchinfo_rating_color('important')).to eq('red')
```

Comment:
Why not testing the other cases?
(At least I would test the case when nothing is found in the hash)